python/llm/example/CPU/Application/streaming/streaming_llm/enable_streaming_llm.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ===========================================================================
+#
+# This file is copied from
+# https://github.com/mit-han-lab/streaming-llm/blob/main/streaming_llm/enable_streaming_llm.py
+# which is licensed under the MIT license:
+#
```

Comment:
You can change to "This file is adapted from xxxx" since there are some modification of importing  'enable_llama_pos_shift_attention' etc.

Reply:
Do we need to change anything in this file?

Reply:
Are you changing this file only to change the flatten the folder structure? I think we can keep the original folder structure of streaming_llm and highlight what we have changed (AutoModelForCasualLM.from_pretrained).